Connection/Connection.php

```diff
@@ -147,11 +146,11 @@
     protected $_remote        = false;
 
     /**
-     * Remote address.
+     * List of Remotes address.
      *
      * @var string
      */
-    protected $_remoteAddress = null;
+    protected $_remotesAddress = [];
```

Comment:
Yes, it's something possible, maybe much appropriate, I need to test if this may have issues related to a pool of nodes.